### PR TITLE
Adding the AIT DSN plugin into the deployment of the AIT-Server

### PR DESCRIPTION
**Pull request by vivekkatial**
Updated the `bootstrap.sh` shell script and the `config` files. 

I've copied the same structure as the installation for the AIT-GUI plugin.

For the config file in `configs/ait/config/config.yaml`, I've used the default `AIT-DSN` parameters based on the configuration recommended by Kabir, available [here](https://github.com/NASA-AMMOS/AIT-DSN/blob/master/config/config.yaml)

### CODE REVIEW

File: configs/ait/config/config.yaml
Comment (michaelfedell) [inline configs/ait/config/config.yaml:50]: Have we looked into what these do? Is there a value we should be injecting based on their deployment?

Comment (michaelfedell) [reply configs/ait/config/config.yaml:50]: same for below sections (`raf`, and `fcltu`)

File: configs/ait/config/config.yaml
Comment (michaelfedell) [inline configs/ait/config/config.yaml:94]: Have you verified that these paths are valid in our deployment?

File: scripts/ait_bootstrap.sh
Comment (michaelfedell) [inline scripts/ait_bootstrap.sh:95]: Let's go ahead and add a `cd $PROJECT_HOME` here once we're done with the git operations

File: templates/ammos-cubs-ait.template.yaml
Comment (michaelfedell) [inline templates/ammos-cubs-ait.template.yaml:224]: good job duplicating the addition here from bootstrap.sh - technically that sh file isn't used yet but we will want to swap over to it in the next round of improvements

**Review by michaelfedell — CHANGES_REQUESTED** [review]
Couple of questions left, but mainly just want to make sure that we have tested this in our deployment and that the default config here makes sense and doesn't break things. We also will need to add some content to our Deployment guide (see `docs/partner_editable`) concerning this new plugin

File: configs/ait/config/config.yaml
Comment (vivekkatial) [reply configs/ait/config/config.yaml:94]: They wont be valid, so we would have to create them -- where about do i create them?

File: configs/ait/config/config.yaml
Comment (vivekkatial) [reply configs/ait/config/config.yaml:50]: These are different standards for space link extensions. Some docs (here)[https://public.ccsds.org/Pubs/911x2b3.pdf] -- I imagine this is what the person using the guide would specify?

File: configs/ait/config/config.yaml
Comment (michaelfedell) [reply configs/ait/config/config.yaml:94]: Let's ask kabir what these are used for

File: configs/ait/config/config.yaml
Comment (michaelfedell) [reply configs/ait/config/config.yaml:50]: Need to document in the deployment guide that these defaults are for reference and are not functional - all the SLE configuration will be left to the user as a post-deployment step

File: configs/ait/config/config.yaml
Comment (michaelfedell) [reply configs/ait/config/config.yaml:94]: We will create these in the AIT bootstrap scripts (`mkdir -p ...`)

**Review by michaelfedell — APPROVED** [review]

**Review by michaelfedell — APPROVED** [review]
LGTM assuming final deployment test goes well.
Need to expand deployment guide separately ⚠️:memo:

**Review by andrew-glenn — APPROVED** [review]
No concerns, I defer to Michael's feedback.

**Pull request merged**
